packages/label-sync/src/labels.json

```diff
@@ -62,7 +62,7 @@
     },
     {
       "name": "priority: p0",
-      "description": "Highest priority. Critical issue. P0 implies highest priority (e.g. blocking imminent release). P0 issues should be monitored on a daily basis.",
```

Comment:
GitHub had a limit on the length of label description @JustinBeckwith; awesome debugging @sofisl.

I believe this was the error that we were seeing when we tried to create labels in parallel with `Promise.all`, I still think for the time being this approach is reasonable however, it's helped us debug a couple issues more easily to run in series.